lib/liquid/block.rb

```diff
@@ -1,9 +1,16 @@
 module Liquid
   class Block < Tag
-    IsTag             = /^#{TagStart}/o
```

Comment:
You seem to be working off a really old commit.  See how different the file is now: https://github.com/Shopify/liquid/blob/master/lib/liquid/block.rb